.github/workflows/ci.yml

```diff
@@ -24,7 +24,6 @@ jobs:
     env:
       MAKEFLAGS: -j2
       PLATFORM: x86_64-linux
-      TAGLIB_VERSION: 1.11.1
```

Comment:
@robinst I have added this change since your first review. It resolves a major source of confusion for me: CI behaving different from local development.

Reply:
OK this breaks CI, never mind for now.